src/main/java/net/boomerangplatform/model/WorkflowQuotas.java

```diff
@@ -0,0 +1,21 @@
+package net.boomerangplatform.model;
+
+public class WorkflowQuotas {
+
+  private Integer maxWorkflowCount;
+  private Integer currentWorkflowCount;
+  
+  public Integer getMaxWorkflowCount() {
+    return maxWorkflowCount;
+  }
+  public void setMaxWorkflowCount(Integer maxWorkflowCount) {
+    this.maxWorkflowCount = maxWorkflowCount;
+  }
```

Comment:
maxWorkflowCount should be on TeamWorkflowSummary since it extends FlowTeamEntity. Do we need maxWorkflowCount on WorkflowQuotas also?